eng/helix/content/runtests.sh

```diff
@@ -11,21 +11,7 @@ YELLOW="\033[0;33m"
 MAGENTA="\033[0;95m"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# Ensures every invocation of dotnet apps uses the same dotnet.exe
-# Add $random to path to ensure tests don't expect dotnet to be in a particular path
-export DOTNET_ROOT="$DIR/.dotnet$RANDOM"
-
-# Ensure dotnet comes first on PATH
-export PATH="$DOTNET_ROOT:$PATH:$DIR/node/bin"
-
-# Prevent fallback to global .NET locations. This ensures our tests use the shared frameworks we specify and don't rollforward to something else that might be installed on the machine
-export DOTNET_MULTILEVEL_LOOKUP=0
-
-# Avoid contaminating userprofiles
-# Add $random to path to ensure tests don't expect home to be in a particular path
-export DOTNET_CLI_HOME="$DIR/.home$RANDOM"
```

Comment:
Likely still need this environment variable since some tests install tools.

Reply:
We use toolpath so I think we can keep this removed if that's the only reason to set it, 'C:\h\w\AD69099C\p\dotnet-cli/dotnet tool install dotnet-ef --version 6.0.0-preview.4.21201.3 --tool-path C:\h\w\AD69099C\w\B9FE09F1\e'

Reply:
As long as the environment is set up mostly consistently between runtests.ps1 and runtests.sh and builds work, sounds good.